packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/selectors.ts

```diff
@@ -14,13 +18,58 @@ export const getData = (state: RootState, { side }: OwnProps) => {
   const coins = selectors.components.swap.getCoins()
   const accounts = getCoinAccounts(state, { coins, ...SWAP_ACCOUNTS_SELECTOR })
   const pairs = selectors.components.swap.getPairs(state).getOrElse([])
-  let coinsForSide
+  const values = selectors.form.getFormValues('initSwap')(state) as InitSwapFormValuesType
+  const custodialEligibility = selectors.components.swap
+    .getCustodialEligibility(state)
+    .getOrElse(false)
 
   if (!pairs.length) return { accounts }
 
+  const _checkBaseCustodial = (account: SwapAccountType) => {
```

Comment:
is this method name a convention for the internal or private methods?

Reply:
Not a convention just wanted to make it clearer that these are helper methods inside the selector